content/packages-and-modules/getting-packages-from-the-registry/resolving-eacces-permissions-errors-when-installing-packages-globally.mdx

```diff
@@ -61,7 +61,7 @@ To minimize the chance of permissions errors, you can configure npm to use a dif
 Instead of steps 3-5, you can use the corresponding ENV variable (e.g. if you don't want to modify `~/.profile`):
```

Comment:
```suggestion
Instead of steps 3-6, you can use the corresponding ENV variable (e.g. if you don't want to modify `~/.profile`):
```

This suggestion includes step 6 too now.

Reply:
Thank u